style.css

```diff
@@ -291,3 +413,268 @@ footer {
+    width: 100px;
+    height: 100px;
+    border-radius: 50%;
+    background-color: #edecff;
+    box-shadow: 0 0 15px 0 #edecff;
+    filter: blur(10px);
+  }
+
+  .intro h2 {
+    font-family: Roboto, Arial, sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 40px;
+    line-height: 47px;
+    text-align: center;
+    color: #dfe1e6;
+    margin-top: -50px;
+  }
+
+  .intro p {
+    margin-right: 30%;
+    margin-left: 30%;
+    color: #b3bac5;
+  }
+
+  .firsthr {
+    margin-left: 30%;
+    margin-right: 30%;
+  }
+
+  .intro p::before {
+    content: '';
+    position: absolute;
+    right: 0;
+    width: 200px;
+    height: 200px;
+    border-radius: 100%;
+    background-color: #fff;
+    box-shadow: 0 0 95px 0 #d1cfff;
+    filter: blur(10px);
+  }
+
+  .abilities {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: 25px;
+    margin-left: 30%;
+    margin-right: 20%;
+  }
+
+  .Framworks::before {
+    display: none;
+  }
+
+  .title {
+    font-family: Roboto, Arial, sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 18px;
+    color: #9390ff;
+  }
+
+  .mywork1 {
+    display: block;
+  }
+
+  .social-media-myWork {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 106px 142px 237px 142px;
+  }
+
+  .projects {
+    margin-left: 142px;
+    margin-right: 142px;
+  }
+
+  .recent_work {
+    font-family: Roboto, arial, sans-serif;
+    font-size: 64px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 114px;
+    text-align: left;
+  }
+
+  .project {
+    display: flex;
+    align-items: flex-end;
+  }
+
+  .preview {
+    width: 800%;
+    height: 400px;
+  }
+
+  .projects hr::after {
+    content: '';
+    position: absolute;
+    width: 57.53px;
+    height: 700.92px;
+    left: -132px;
+    top: 1479.12px;
+    background: #edecff;
+    filter: blur(50px);
+    transform: rotate(-36.38deg);
+  }
+
+  .comment-section {
+    margin: 50px 40px 0 40px;
+  }
+
+  .first::after {
+    display: none;
+  }
+
+  .second {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  .second::after {
+    content: '';
+    position: absolute;
+    width: 108px;
+    height: 402px;
+    right: 0;
+    top: 2257.74px;
+    background: #edecff;
+    filter: blur(50px);
+    transform: matrix(-0.84, -0.54, -0.54, 0.84, 0, 0);
+  }
+
+  .third2::before {
+    display: none;
+  }
+
+  .third::after {
+    content: '';
+    position: absolute;
+    width: 308px;
+    height: 155px;
+    left: -41px;
+    background: #edecff;
+    filter: blur(20px);
+  }
+
+  .f4::before {
+    display: none;
+  }
+
+  .fourth {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  .fourth::before {
+    content: '';
+    position: absolute;
+    width: 146.19px;
+    height: 616.38px;
+    right: 0;
+    background: #edecff;
+    filter: blur(50px);
+    transform: matrix(-0.26, 0.97, 0.97, 0.26, 0, 0);
+  }
```

Comment:
- [ ] Kindly when you added the images there is an overflow in the design.

![image](https://user-images.githubusercontent.com/35535130/154744545-409cab7d-e260-45d3-9064-214e15980d87.png)

![image](https://user-images.githubusercontent.com/35535130/154744618-fe4df3e8-0e0e-49b9-8a84-2228addda7b2.png)